python/lsst/pipe/tasks/associationUtils.py

```diff
@@ -193,3 +193,71 @@ def query_disc(nside, ra, dec, max_rad, min_rad=0):
         pixels = pixels[match]
 
     return pixels
+
+
+def objID_to_ssObjectID(objID, flags=0):
```

Comment:
Our naming conventions would write this as
```
obj_id_to_ss_object_id
```
or
```
objIdToSsObjectId
```
(same for other symbols in this file)